src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingVisitor.cs

```diff
@@ -341,7 +341,14 @@ public override void VisitMarkupAttributeBlock(MarkupAttributeBlockSyntax node)
 
     public override void VisitMarkupTagHelperAttribute(MarkupTagHelperAttributeSyntax node)
     {
-        Visit(node.Value);
+        WriteBlock(node, FormattingBlockKind.Tag, n =>
```

Comment:
This updates tag helper attributes to match the logic of html tag attributes, in the method above.

I don't fully understand:

* What this is doing
* Why it wasn't doing it before
* Why only one of the tests was fixed with this change
* Why none of the existing tests broke with this change

Formatting! :)